echopype/convert/api.py

```diff
@@ -168,20 +169,20 @@ def _save_groups_to_file(echodata, output_path, engine, compress=True):
             group=f"Sonar/{BEAM_SUBGROUP_DEFAULT}",
             compression_settings=COMPRESSION_SETTINGS[engine] if compress else None,
         )
-    if echodata.beam_power is not None:
-        io.save_file(
-            echodata.beam_power.chunk(
-                {
-                    "range_sample": DEFAULT_CHUNK_SIZE["range_sample"],
-                    "ping_time": DEFAULT_CHUNK_SIZE["ping_time"],
-                }
-            ),
-            path=output_path,
-            mode="a",
-            engine=engine,
-            group="Sonar/Beam_group2",
-            compression_settings=COMPRESSION_SETTINGS[engine] if compress else None,
-        )
+        if echodata.beam_power is not None:
+            io.save_file(
+                echodata.beam_power.chunk(
+                    {
+                        "range_sample": DEFAULT_CHUNK_SIZE["range_sample"],
+                        "ping_time": DEFAULT_CHUNK_SIZE["ping_time"],
+                    }
+                ),
+                path=output_path,
+                mode="a",
+                engine=engine,
+                group="Sonar/Beam_group2",
+                compression_settings=COMPRESSION_SETTINGS[engine] if compress else None,
+            )
```

Comment:
Interesting, so the logic before was actually not correct, and it ran without problem just because `echodata.beam_power` only exist for EK80?

Reply:
`echodata.beam_power` aliases to the second beam group I think, even with AD2CP. Previously it wouldn't save the 3rd or 4th beam group

Reply:
That if statement is not necessary for AD2CP because every beam group will be saved in the for loop that I wrote above